### DISCUSSION TITLE
I cannot use 'Custom Field Choice Sets' option

### DISCUSSION THREAD

**Discussion by wypxl** [netbox-community/netbox#13733] [General]
Hi, everyone, Im new guy here.
I recently installed NetBox 3.6.0, I am still relatively unfamiliar with this system, and the operation is basically based on the WEB. When I tried to customize a 'choice set', i found that i cannot find the added option on the web interface. Can someone tell me how to customize a 'choice set'? Thanks a lot.

**Comment by candlerb** [comment]
What did you try? What error did you see, or what was missing that you expected to see? Paste in a screenshot if necessary.

For me, I simply go to "Customization > Custom Field Choices" in the left-hand navigation.

To create a new choice set, click the green "Add" button at the top right. To edit an existing Choice Set, first click on its name to get to the detail page, then click the yellow Edit button.

![image](https://github.com/netbox-community/netbox/assets/44789/657fc53d-1d81-46ca-a91b-47556a874418)

![image](https://github.com/netbox-community/netbox/assets/44789/8f898518-8a72-406c-80b8-5cfb3126d438)

**Reply by wypxl** [reply]
Hi， its my fault, I ingore that i had changed menu.py. That is why i could not find the added option. Thank you for your patience and help.

**Reply by candlerb** [reply]
OK, good. Best not to do that then :-) [Plugins](https://docs.netbox.dev/en/stable/plugins/) provide a supported way to customize Netbox, including adding to the main navigation menu.
